taxi/forms.py

```diff
@@ -0,0 +1,17 @@
+from django import forms
+from .models import Car, Manufacturer
+from crispy_forms.helper import FormHelper
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = "__all__"
+    helper = FormHelper()
+
+
+class ManufacturerForm(forms.ModelForm):
+    class Meta:
+        model = Manufacturer
+        fields = "__all__"
+    helper = FormHelper()

```

Comment:
Similar to the `CarForm`, the `helper` attribute in `ManufacturerForm` should be placed inside the `__init__` method to ensure proper initialization for each form instance.